### ISSUE TITLE
[AVT] Accessible name must match or contain the visible label text

### ISSUE THREAD

**Issue by yanweibj** [carbon-design-system/carbon#5402]
## Environment

> Operating system
MacOS
> Browser
Chrome
> Automated testing tool and ruleset
DAP
IBM Accessibility January 2020
IBM Accessibility Experimental

## Detailed description

> What version of the Carbon Design System are you using?
Carbon v10
> What did you expect to happen?
Accessible name should contain the visible text label
> What happened instead?
Accessible name does not contain the visible text label
https://aat.w3ibm.mybluemix.net/token/dd9b1ac8-55f6-4f23-a8fa-9b4851c53e38/fafaad8a-304a-497f-bcaa-f64496bff7bf/archives/2019NovDeploy//doc/w3/help/en-US/idhi_accessibility_check_g1202.html
> What WCAG 2.1 checkpoint does the issue violate?
2.5.3 Label in Name

## Steps to reproduce the issue

1.  Render a Dropdown component
2.  In inspect check that accessible name does not contain the visible text label

**Comment by yanweibj** [comment]
<img width="637" alt="image" src="https://user-images.githubusercontent.com/16932498/75252282-57473d00-5817-11ea-80c0-a3e05a4c22b9.png">

**Comment by yanweibj** [comment]
<img width="639" alt="image" src="https://user-images.githubusercontent.com/16932498/75252354-7c3bb000-5817-11ea-8a59-3641bd6ea96d.png">

**Comment by asudoh** [comment]
Seems a duplicate of #4531, which is addressed by #4820.
